README.md

```diff
@@ -227,18 +230,33 @@ analyze(results)
 
 ## Generate the documentation
```

Comment:
Shouldn't we move it to CONTRIBUTING document?
I guess that's not needed for a regular user (not to expand README too much)

Reply:
Yes, there is no need to explain how to build the doc here